### PR TITLE
Add "roadies" package

### PR DESCRIPTION
**Pull request by ang037**
ROADIES is a novel pipeline designed for phylogenetic tree inference of the species directly from their raw genomic assemblies. ROADIES offers a fully automated, easy-to-use, scalable solution, eliminating any error-prone manual steps and providing unique flexibility in adjusting the tradeoff between accuracy and runtime.

For more details, please refer to the ROADIES [GitHub repo](https://github.com/TurakhiaLab/ROADIES) and [Wiki](https://turakhia.ucsd.edu/ROADIES/).

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/roadies/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    - {{ pin_subpackage("roadies", max_pin=None) }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - wget
+    - unzip
+    - libboost-devel
```
Comment (martin-g) [inline recipes/roadies/meta.yaml:23]: liboost, zlib and Python should be removed from 'build' deps

Comment (ang037) [reply recipes/roadies/meta.yaml]: Done as requested

**Comment by ang037** [comment]
@BiocondaBot please add label

File: recipes/roadies/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/TurakhiaLab/ROADIES/archive/v{{ version }}.tar.gz
+  sha256: e3b4b89f247824c4b61e790f094320624bde47e295e2158a9ad83c05dfa83d54
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("roadies", max_pin=None) }}
```
Comment (martin-g) [inline recipes/roadies/meta.yaml:14]: Are you sure about 'None' here?
See the Pull request template for its meaning.

Comment (ang037) [reply recipes/roadies/meta.yaml:14]: Thanks for bringing this up. We are working on updating our repo to support semantic versioning.

**Review by martin-g — APPROVED** [review]

**Pull request merged**